content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -3,22 +3,20 @@ title: Setting up Oracle
 kind: documentation
 description: Setting up Database Monitoring on an Oracle database
 disable_sidebar: true
-private: true
-is_beta: true
 ---
 
 {{< site-region region="gov" >}}
 <div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
 {{< /site-region >}}
 
 <div class="alert alert-info">
-The features described on this page are in private beta.
+The features described on this page are in beta. Contact your Customer Success Manager to provide feedback or ask for help.
 </div>
 
 ## Supported Oracle versions, features, and architectures
 
 - **Versions**: 19c and 21c
-- **Deployment configurations**: Self-managed, RDS, RAC, Exadata
+- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous database
```

Comment:
Should the d in "database" be capitalized?

Reply:
I'll update this one as well.